controller/array_action/array_mediator_ds8k.py

```diff
@@ -617,32 +618,45 @@ def get_iscsi_targets_by_iqn(self, host_name):
 
     def get_array_fc_wwns(self, host_name):
         logger.debug("getting the connected fc port wwpns for host {} from array".format(host_name))
+        api_host = self._get_api_host_by_name(host_name)
+        wwpns = [port[LOGIN_PORT_WWPN] for port in api_host.login_ports if
```

Comment:
I know it shouldn't happen, but still, if `api_host` is `None` here, I think we should log it and return an empty list or something, instead of getting an unindicative "`NoneType` has no `login_ports` attribute"

Reply:
done

Reply:
done